.github/workflows/enarx-standard.yml

```diff
@@ -4,9 +4,25 @@ name: Enarx Standard Tests
 on: [pull_request]
 
 jobs:
+  # Ensure the testing infrastructure is working as intended.
   sanity-check:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@master
+    - uses: actions/checkout@v1
```

Comment:
If there's a reason to change this, it should be an independent logical commit with clear explanation.

Reply:
You caught me realizing this was a tagging system in real time :stuck_out_tongue: